Passwordscript.js

```diff
@@ -0,0 +1,56 @@
+var myInput = document.getElementById("psw");
+var myInput2 = document.getElementById("psw2");
+var letter = document.getElementById("letter");
+var capital = document.getElementById("capital");
+var number = document.getElementById("number");
+var length = document.getElementById("length");
+var specialDigit = document.getElementById("specialDigit");
```

Comment:
Jeżeli korzystasz z czegoś tylko w ramach jednej funkcji, to deklaruj to jako zmienne lokalne, a nie zmienne w kodzie globalnym. Do deklaracji wykorzystuj let i const zamiast var